packages/hub/node-tests/helpers/cards.ts

```diff
@@ -7,12 +7,14 @@ import { CardService, INSECURE_CONTEXT } from '../../services/card-service';
 import FileCache from '../../services/file-cache';
 import { TEST_REALM } from '@cardstack/core/tests/helpers';
 import { RealmConfig } from '@cardstack/core/src/interfaces';
-import { BASE_REALM_CONFIG, DEMO_REALM_CONFIG } from '../../services/realms-config';
+import { BASE_REALM_CONFIG } from '../../services/realms-config';
 
 tmp.setGracefulCleanup();
 
+let fileCacheTmpDir = tmp.dirSync();
```

Comment:
This change, along with the removal of the cache-dir cleanup on Container teardown allows us to use previously generated ComponentMeta files. This is because our file-cache has optimizations so that we do not re-index a realm if the files haven't changed. The problem is, we clear the cache between each test, and the newly generated meta modules are not being recreated.